src/delete.ts

```diff
@@ -180,3 +35,58 @@ import ProductInterface from "./models/models";
 
 	await browser.close();
 })();
+
+async function getProducts() {
+	return fetch(
+		`${process.env.ELASTIC_URL}/products/_search` ||
+			"https://localhost:9200/products/_search",
+		{
+			method: "post",
```

Comment:
Should this getProducts not be a get method instead of post?

Reply:
No, it needs a body